tensorboard/plugins/interactive_inference/utils/inference_utils.py

```diff
@@ -531,30 +540,41 @@ def make_json_formatted_for_single_chart(mutant_features,
         key = classification_class.label
         if index_to_mutate:
           key += ' (index %d)' % index_to_mutate
-        series[key].append({
-            x_label: mutant_feature.mutant_value,
-            y_label: classification_class.score,
-        })
+        if not key in series:
+          series[key] = {}
+        if not mutant_feature.mutant_value in series[key]:
+          series[key][mutant_feature.mutant_value] = []
+        series[key][mutant_feature.mutant_value].append(classification_class.score)
```

Comment:
this line seems to be over the width limit

Reply:
fixed